pyronear/models/ssresnet18.py

```diff
@@ -0,0 +1,78 @@
+from torchvision.models.resnet import ResNet, BasicBlock
+import torchvision
+from torch import nn
+import torch
+import torch.nn.functional as F
+
+
+class SSResNet18(ResNet):
+    def __init__(self, frame_per_seq):
+        super(SSResNet18, self).__init__(BasicBlock, [2, 2, 2, 2])
```

Comment:
For your model to be easily reusable, I would suggest rather creating a `SSResNet` class, and have a `ssresnet18` function that will return the correct layer layout (similarly to what is done in [torchvision](https://github.com/pytorch/vision/blob/master/torchvision/models/resnet.py#L232)